src/graphql/node.rs

```diff
@@ -12,13 +12,17 @@ use async_graphql::{
 use bincode::Options;
 use chrono::{DateTime, TimeZone, Utc};
 #[allow(clippy::module_name_repetitions)]
-pub use crud::get_customer_id_of_node;
+pub use crud::agent_keys_by_customer_id;
 use database::Indexable;
 use input::NodeInput;
 use review_database as database;
 use roxy::Process as RoxyProcess;
 use serde::{Deserialize, Serialize};
 
+const SENSOR_AGENT: &str = "piglet";
+const UNSUPERVISED_AGENT: &str = "reconverge";
+pub(super) const SEMISUPERVISED_AGENT: &str = "hog";
```

Comment:
Since "semi-supervised" is an official term, we should use `SEMI_SUPERVISED`.

Reply:
done